source/indexes/atlas-search-index.txt

```diff
@@ -114,7 +142,10 @@ Create a Search Index with Dynamic Mappings
 Use dynamic mappings in your Atlas Search index to automatically index all fields of supported types.
 For a list of supported BSON data types, see the :atlas:`Data Types </atlas-search/define-field-mappings/#data-types>` section in the Atlas Search documentation. 
 
-To create a single Atlas Search index with dynamic mappings, call the ``create_one()`` method and set ``mappings.dynamic`` to ``true``. 
+To create a single Atlas Search index with dynamic mappings, first create a ``definition`` document containing the ``mappings`` field. 
```

Comment:
Q: Same question as above

Reply:
Hmm if I commit to the idea that an index uses mappings, then I can change it to "To create a single Atlas Search index _that uses_ dynamic mappings."

Reply:
^ Q: Should I update the section header to reflect this?

Reply:
not necessary